recipe/meta.yaml

```
@@ -23,42 +20,42 @@ requirements:
     - setuptools
     - pip
     - wheel
```

```suggestion
    - pip
```
pip and setuptools are not needed when a package uses a different build backend than setuptools.